additionalalerts.xml

```diff
@@ -6,10 +6,12 @@
     <logo>https://raw.githubusercontent.com/InfotelGLPI/additionalalerts/master/additionalalerts.png</logo>
     <description>
         <short>
+            <cs><![CDATA[Posílání dalších upozornění. Tento zásuvný modul umožňuje posílat upozornění e-mailem ohledně nových strojů v OCS-NG, nezinventarizované počítače za X dnů, položky s nevyplněným datem nákupu a náplních do tiskáren, které docházejí.]]></cs>
             <fr><![CDATA[Envoi d'alertes supplémentaires. Ce plugin vous permet d’envoyer les alertes email concernant les nouvelles machines inventoriés depuis OCS-NG, les machines non inventoriés depuis X jours, les matériels ayant une date d'achat vide et les cartouches ayant un niveau faible.]]></fr>
             <en><![CDATA[Sending of additional alerts. This plugin enables you to send email alerts regarding the new machines inventoried from OCS-NG, the not inventoried computers from X days, the items with empty buy date and Cartridges whose level is low.]]></en>
         </short>
         <long>
+            <cs><![CDATA[Tento zásuvný modul umožňuje posílat e-mailem doplňková upozornění:<br />- Nové stroje zinventarizované v OCS-NG (mající stav určený do nastavovacího modulu OCS-NG),<br />- Počítače nezinventarizované za X dnů (Volba podporovaného stavu: aby nebyly brány v úvahu stroje na skladě),<br />- Položky s nevyplněným datem zakoupení (Volba podporovaných typů: aby nebylo v úvahu bráno vybavení, které ještě nebylo zakoupeno),<br />- Náplně do tiskáren, které docházejí (FusionInventory).]]></cs>
```

Comment:
Sorry, suddenly started to place it only alphabetically. Then realized, that it is driving order of panels with description in pages on https://plugins.glpi-project.org site. So, switched to schema where tag with description is en first (because rest of displayed page is), then by alphabetical order, to easier orientation for reader/editor. With tags lang and keywords then alphabetically only, because they seems are not influencing anything.
Is this correct please?

Reply:
The new languages must be added afterwards to respect the order of all the plugins so <fr>, <en> and the new languages

Reply:
Must admit, that I cannot understand the rest of sentence after word afterwards.
What is meant with "order of all plugins" in this case context?

Please, what is wrong with idea, that on plugins.glpi-project.org, there tabs with languages of description should be in alphabetical order, except for English (to display by default because rest of page is in English).

And order in list of supported languages should be alphabetical too (see for fr_FRen_GBde_DEes_ESit_ITpl_PLpt_BRru_RUtr_TRes_ARet_EEfi_FIhe_ILpt_PTsl_SIzh_TW in https://plugins.glpi-project.org/#/plugin/fusioninventory – it is very difficult for reader to locate her/his language without resorting to browser search function).

See, I don't want make problems, I will place description, language codes and tags in whatever order you want in xml data, but I'm unhappy about resulting bad experience for plugins.glpi-project.org visitors, which is resulting from not doing alphabetical ordering in presenting information to them.

I want to continue to work on translating of descriptions for another plugins for plugins.glpi-project.org, so please lets try to resolve this issue (maybe open discussion across team to have some general resolution?).

Thanks :)
Pavel

Reply:
The modifications we asked you to make are not related to tags.
You made no modification to change this order: 
![translate](https://user-images.githubusercontent.com/15138371/44339211-d9b3ab80-a480-11e8-9af5-4d53453dabd7.png)

The change we ask you is the order of the tabs : 
![onglet](https://user-images.githubusercontent.com/15138371/44339259-fcde5b00-a480-11e8-84b0-3fcf9df8797c.png)

Reply:
You just have to add after those already existing as you did here: https://github.com/InfotelGLPI/webapplications/commit/af702b9063d252aa78f8fc44b42fc8c75b38defb

Reply:
Well, I do see – but result is, that visitor is reading English page text, then suddenly there is some text in french, then (s)he cannot locate desired language, because order is not alphabetical, the same on some plugins pages for supported languages and result is bad user experience only because unfortunate presentation of existing data. This is point I tried to improve along the adding localization too…